STANDARD.md

```diff
@@ -248,11 +248,19 @@ In order to correlate openPMD iterations with an index of data-format internal u
     - advice to implementers: an openPMD iteration might be spread over multiple updates/steps, but not vice versa.
                               In such a scenario, an individual openPMD record's update/step must appear exactly once per iteration.
 
+Notes:
+
+* In implementations without support for IO steps, the variable-based encoding of iterations may still be used for storage of a single iteration.
```

Comment:
In these points, the new wording using `IO steps` might be confusing?

Reply:
I've reworded that to `support for storing multiple versions of datasets/attributes`, which I think is more explicit anyway.